frontend-web/webclient/app/Applications/Jobs/View.tsx

```diff
@@ -450,9 +458,13 @@ const Busy: React.FunctionComponent<{job: Job}> = ({job}) => {
     return <BusyWrapper ref={ref}>
         <Box mt={"16px"}>
             <Box mb={"16px"}>
-                Your reserved machine is currently quite popular.
-                Cluster utilization is currently at {clusterUtilization}% with {numberOfJobs} jobs running
-                and {numberOfJobsInQueue} in the queue.
+                Your reserved machine is currently quite popular.<br />
```

Comment:
In case that utilization isn't large, it would make sense that we simply display a message about preparing the software since this step might also be a reason that it is slow.

Reply:
Good idea. So if the cluster utilization is less than 80%? Or should it be higher/lower?